### ISSUE TITLE
height of inputToolbar

### ISSUE THREAD

**Issue by wdcurry** [jessesquires/JSQMessagesViewController#1070]
Just a thought. The current default makes no sense, as the user can easily hit Return over and over and obliterate the view. It is easily set of course, to a random height with 
`self.inputToolbar.maximumHeight = 100;`

On a related note: if i ram text into the textview (think of a canned msg), what is the currently accepted practice to resize the textview?

**Comment by wdcurry** [comment]
There may be a better (already set) way to get'r'done, but for now, i moved these from the .m to the .h to make them accessible to the instance of the message viewer:
```
- (void)jsq_adjustInputToolbarForComposerTextViewContentSizeChange:(CGFloat)dy;
- (void)jsq_adjustInputToolbarHeightConstraintByDelta:(CGFloat)dy;
- (void)jsq_updateCollectionViewInsets;
```
{edits to JSQMessagesViewController.h/.m}
.. while this is leaving one errant line too long, for now it is far better than a single line that makes reaching deeper by scrolling into the code-entered text much easier.

**Comment by jessesquires** [comment]
hey @wdcurry - not sure i'm following you here...

**Comment by wdcurry** [comment]
imagine firing up a chat view, and within code, entering text into the textView, which enables the user to tap send immediately. If this text is more than 1 line high, the textview needs adjustment to fit. Out-of-the-box, it is only 1-line tall, and trying to finger-focus the caret is very very challenging. The above code, allows us to call each from our descendent, allows the textview to resize.

**Comment by jessesquires** [comment]
Ahh, you should be able to post [`UITextViewTextDidChangeNotification`](https://developer.apple.com/library/prerelease/ios/documentation/UIKit/Reference/UITextView_Class/index.html#//apple_ref/c/data/UITextViewTextDidChangeNotification)

**Comment by KeyzenNain** [comment]
Hi, @jessesquires, related to this issue. I found the similar issue. The height wasn't resized automatically although the notification worked (JSQMessagesComposerTextView's jsq_didReceiveTextViewNotification was called.)
In the sample project, it worked fine. However my project is not. The difference I found was the deployment target. My project has 8.1 and the sample has 7. I tried to figure out what was the root cause. But I've not got any idea so far. 
Any clue?
